sql/sql_expression_operations.cc

```diff
@@ -2256,21 +2256,48 @@ bindBuiltinFunction(SqlBindingScope & scope,
                 this,
                 fn.resultInfo};
     }
-    else {
+
+    if (functionName == "try") {
```

Comment:
This is the wrong place to put it.  We shouldn't have logic hardcoded into the generic function wrapper here.